spec/Schema.spec.js

```diff
@@ -494,7 +494,8 @@ describe('SchemaController', () => {
 
   it('creates non-custom classes which include relation field', done => {
     config.database.loadSchema()
-    .then(schema => schema.addClassIfNotExists('_Role', {}))
```

Comment:
That's what I was expecting as well, but I was getting `Class _Role already exists.`